### PR TITLE
update

### PR DESCRIPTION
**Pull request by Sivakumar-21**

### CODE REVIEW

File: app/Models/Client.php [hunk 1]
```diff
@@ -19,8 +19,11 @@ public function clientcontacts()
         return $this->hasMany(Clientcontact::class);
     }
 
-    public static function getsupplier(){
-        return self::where('is_supplier', true)->select('id','name')->get();
+    public static function getSupplier()
+    {
+        return self::where('is_supplier', true)
```
Comment (muni2explore) [inline app/Models/Client.php:24]: where order by

File: app/Http/Controllers/ProjectController.php [hunk 2]
```diff
@@ -83,9 +83,9 @@ public function show(Project $project)
     // public function edit($id)
     public function edit(Project $project)
     {
-        $categories = ProjectCategory::select('id', 'name')->get();
-        $statuses = ProjectStatus::select('id', 'name')->get();
-        $clients = Client::select('id', 'name')->get();
+        $categories = ProjectCategory::getProjectCategory();
```
Comment (muni2explore) [inline app/Http/Controllers/ProjectController.php:86]: reneme getProjectCategories

File: app/Models/Client.php (hunk 1 above)
Comment (muni2explore) [reply app/Models/Client.php:24]: getSuppliers

**Review by muni2explore — CHANGES_REQUESTED** [review]

File: app/Models/Client.php (hunk 1 above)
Comment (Sivakumar-21) [reply app/Models/Client.php:24]: changed

File: app/Http/Controllers/ProjectController.php (hunk 2 above)
Comment (Sivakumar-21) [reply app/Http/Controllers/ProjectController.php:86]: changed

**Review by muni2explore — APPROVED** [review]
Looks good

**Pull request merged**